build-in-container/README.md

```diff
@@ -14,6 +14,9 @@ The mariner-docker-builder.sh script presents these options <br />
 
 Optional arguments <br />
   --mariner_dir             directory to use for Mariner artifacts (SPECS, toolkit, ..). Default is the current directory <br />
+  --RPM_repo_file           Path to a custom repo file. Please see [here](./README.md#sample-custom-repo) for sample custom repo file  <br />
```

Comment:
Not critical right now, but to fully support the tools we should support a list of repo files.